src/apps/CMakeLists.txt

```diff
@@ -15,3 +15,12 @@ endif(WIN32)
 
 add_executable(bsqlodbc bsqlodbc.c)
 target_link_libraries(bsqlodbc tdsodbc replacements tdsutils ${libs})
+
+if(WIN32)
+INSTALL(TARGETS tsql bsqlodbc
```

Comment:
quite lot of repetitions. Maybe defining a install_win32_targets would help

Reply:
There are a lot of repetitions. I would like to do only one install clause, but (At least as far as I know) cmake doesn't allow to install targets outside each CMakelists.txt.

Reply:
was thinking about defining a "install_targets_win32" macro in the main CMakeLists.txt to avoid all directory repetition so you could use something like install_targets_win32(tdsl bsqlodbc)

Reply:
why only installing on win32 ?

Reply:
For me it's ok defining "install_targets_xxxx" and not write repetitions in each CMakeLists.txt
I just added win32, because it is the platform where I'm testing. 
But the install process can be used in all platforms.